https://gitlab.com/samba-team/samba/-/commit/e13875601ff6f488b48e7de0f6838db3565401d4
https://gitlab.com/samba-team/samba/-/merge_requests/2617
https://gitlab.com/samba-team/samba/-/merge_requests/1025

# Define NETDB_INTERNAL and NETDB_SUCCESS if they are not defined
#
# Gentoo bug 832629 and 835017

--- a/nsswitch/wins.c
+++ b/nsswitch/wins.c
@@ -40,6 +40,14 @@ static pthread_mutex_t wins_nss_mutex = PTHREAD_MUTEX_INITIALIZER;
 #define INADDRSZ 4
 #endif

+#ifndef NETDB_INTERNAL
+#define NETDB_INTERNAL -1
+#endif
+
+#ifndef NETDB_SUCCESS
+#define NETDB_SUCCESS 0
+#endif
+
 _PUBLIC_ON_LINUX_
 NSS_STATUS _nss_wins_gethostbyname_r(const char *hostname,
 				     struct hostent *he,
